pkg/ebpf/tracer.go

```diff
@@ -832,3 +837,12 @@ func (t *Tracer) getProbeProgramIDs() (map[string]uint32, error) {
 	}
 	return fds, nil
 }
+
+func (t *Tracer) conntrackExists(conn *ConnTuple) bool {
+	ok, err := t.conntrack.Exists(conn)
+	if err != nil {
+		log.Errorf("error checking conntrack for connection %+v", *conn)
+	}
```

Comment:
Let's make conntrack errors cause the whole tracer call to fail instead of supressing errors this way. If problems come up with conntrack, we should be aware of them quickly.

Reply:
As @p-lambert pointed out above, this could be because the container goes away mid-flight, although I suspect there is some ref-counting going on in the kernel. But the errors could be only happening in certain connections, and not others, in which case aborting the whole tracer may not be a good idea.

Reply:
We do revert back to old behavior of expiring connections for a connection if we get an error back from conntrack. So if conntrack is wholly borked we'll just revert back to old behavior.